extractor_service/test_upload.py

```
@@ -0,0 +1,6 @@
+import requests
+
+with open(r"C:\Builder\cv_ay.pdf", "rb") as f:
+    files = {"file": f}
+    response = requests.post("http://localhost:8001/extract", files=files)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)